frontend/src/components/ui/data-table.ts

```diff
@@ -73,7 +75,7 @@ export class DataTable extends TailwindElement {
             `,
           )}
         </btrix-table-head>
-        <btrix-table-body>
+        <btrix-table-body class="overflow-scroll">
```

Comment:
I think we only want scrollbars when there's content that overflows the element's size, no? Otherwise there's this extra empty scrollbar space on the bottom and right of the element: 
<img width="482" alt="image" src="https://github.com/webrecorder/browsertrix-cloud/assets/5727389/1b9eb654-42a5-4d83-971b-1a9e2d511333">

```suggestion
        <btrix-table-body class="overflow-auto">
```

Reply:
Well that's dumb, good catch!  Thanks!